lab_1_classify_by_unigrams/main.py

```diff
@@ -3,18 +3,27 @@
 
 Language detection
 """
+
+
 # pylint:disable=too-many-locals, unused-argument, unused-variable
 
 
 def tokenize(text: str) -> list[str] | None:
-    """
+    if isinstance(text, str):
+        tokens = []
+        for symbol in text:
+            if symbol.isalpha():
```

Comment:
use operator continue if symbol is not alpha.
this will help avoid nesting.

Reply:
done